scripts/jenkins/cloud/ansible/roles/deploy_caasp_terraform/tasks/deploy_caasp.yml

```diff
@@ -260,6 +260,13 @@
     skuba_init_command_results
   failed_when: " not ('[init] configuration files written to' in skuba_init_command_results.stdout)"
 
+- name: Management Machine | Update openstack.conf if using Octavia
+  lineinfile:
+    path: "{{ caasp_openstack_path }}/openstack.conf"
+    insertafter: '\[LoadBalancer\]'
+    line: 'use-octavia=true'
```

Comment:
Should this be use_octavia? I thought having export OS_USE_OCTAVIA=true in osrc would be sufficient for octavia calls.

Reply:
No it's use-octavia. In the in-tree openstack cloud provider:
https://github.com/kubernetes/kubernetes/blob/dde6e8e7465468c32642659cb708a5cc922add64/staging/src/k8s.io/legacy-cloud-providers/openstack/openstack.go#L103

Confusingly it defaults to true in the external provider:
https://github.com/kubernetes/cloud-provider-openstack/blob/fa7494f684ebd635648bd3922ce2e71879c790fc/docs/using-openstack-cloud-controller-manager.md#load-balancer

I believe OS_USE_OCTAVIA is only used by Terraform.

Reply:
Yes, isn't this only for Terraform?
Should this then be removed?: https://github.com/SUSE-Cloud/automation/blob/89ee4eb951313c7badaa82c8f3766a5d85d693f2/scripts/jenkins/cloud/ansible/roles/deploy_caasp_terraform/tasks/setup_management_machine.yml#L131

Reply:
My mistake. Just can't find where this is given to k8s. Looks fine after rechecking.